http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/Config.scala

```diff
@@ -454,6 +458,10 @@ object Config {
   }
 
   implicit private val networkTagValueReader: ValueReader[NetworkTag] = stringValueReader.map(NetworkTag)
+  implicit private val containerRegistryUsernameValueReader: ValueReader[ContainerRegistryUsername] =
```

Comment:
nit, maybe rename to `sasRegistry*`?

Reply:
I'm thinking we can reuse the same type for other container registries as well if we have more in the future...it doesn't have to be SAS specific